src/query_parser/v2/parser.y

```diff
@@ -834,6 +841,53 @@ numeric_range(A) ::= LSQB param_num(B) RSQB. [NUMBER]{
   A = NewNumericFilterQueryParam_WithParams(ctx, &B, &B, 1, 1);
 }
 
+/////////////////////////////////////////////////////////////////
+// Numeric Operators
+/////////////////////////////////////////////////////////////////
+
+expr(A) ::= modifier(B) COLON EXCLAMATION EQUAL param_num(C). {
+  QueryParam *D = NewNumericFilterQueryParam_WithParams(ctx, &C, &C, 1, 1);
+  
+  if (D) {
+    // we keep the capitalization as is
+    D->nf->fieldName = rm_strndup(B.s, B.len);
+    QueryNode* E = NewNumericNode(D);
+    A = NewNotNode(E);
+  } else {
+    A = NewQueryNode(QN_NULL);
+  }
```

Comment:
Is this reachable? Seems that `D != NULL` always.

Reply:
It's not reachable. It was removed.